.github/workflows/ci.yml

```diff
@@ -76,6 +76,10 @@ jobs:
           ]
         allow-failure: [false]
         include:
+          - ember-try-scenario: ember-lts-5.12
+            allow-failure: true
+          - ember-try-scenario: ember-6.4
+            allow-failure: true
```

Comment:
These are failing at the moment, but it's likely failing because of issues with the tests themselves. Will fix them up later.

I have a project on ED v5.3 + ember-source 6.4 that uses factory guy, so I am confident the addon itself is OK for scenarios.